paparazzi/build.gradle

```diff
@@ -55,11 +55,10 @@ dependencies {
   api platform(libs.kotlin.bom)
   implementation libs.moshi.core
   implementation libs.moshi.adapters
+  implementation libs.moshi.kotlinReflect
   implementation libs.jcodec.core
   implementation libs.jcodec.javase
 
-  ksp libs.moshi.kotlinCodegen
```

Comment:
hmm, bye-bye KSP Gradle Plugin for :paparazzi and :paparazzi-gradle-plugin?

Reply:
i mean i guess i couldddddd use reflection only for `Snapshot` and codegen'd for the rest, but felt this was cleaner.  Any reason to reconsider?

Reply:
That might be confusing to maintain, one way of serialization is probably enough.
Aren't all serialized classes public? It's the same problem for all.

Btw, my point was that this is the last `ksp` line, so the apply plugin can be removed. Which might totally remove KSP from the build.